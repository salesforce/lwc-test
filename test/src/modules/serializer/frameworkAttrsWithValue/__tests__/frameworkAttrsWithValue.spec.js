/*
 * Copyright (c) 2024, Salesforce, Inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { createElement } from 'lwc';
import FrameworkAttrsWithValue from '../frameworkAttrsWithValue';

it('serializes a component containing framework-supplied attributes', () => {
    const elm = createElement('serializer-component', { is: FrameworkAttrsWithValue });
    document.body.appendChild(elm);

    expect(elm).toMatchSnapshot();
});
