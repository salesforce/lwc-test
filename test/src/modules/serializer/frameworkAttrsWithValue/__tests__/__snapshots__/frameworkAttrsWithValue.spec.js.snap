// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializes a component containing framework-supplied attributes 1`] = `
<serializer-component>
  <h1>
    Hello world
  </h1>
</serializer-component>
`;
